### DISCUSSION TITLE
Can tiktoken version constraint be relaxed?

### DISCUSSION THREAD

**Discussion by cody-moveworks** [openai/whisper#1713] [Q&A] [answered]
The latest release of `openai-whisper` package pins `tiktoken` package to 0.3.1. Can the version constraint be relaxed to allow using more recent versions of the `tiktoken` package? We use both `tiktoken` and `openai-whisper` packages, but due to this constraint we cannot upgrade `tiktoken` package to a more recent version.

**Comment by jongwook** [comment, marked as answer]
Done in #1759!
